README.md

```diff
@@ -19,7 +19,7 @@ This list tries to cover topics on discrete and combinatorial optimization, oper
     1. [Combinatorial Optimization](#combinatorial-optimization)
     1. [Stochastic Optimization](#stochastic-optimization)
 3. [**Solvers, Tools, and Libraries**](#solvers-tools-and-libraries)
-    1. [Licensed/Commercial Solvers](#licensed-commercial-solvers)
+    1. [Licensed/Commercial Solvers](#licensedcommercial-solvers)
```

Comment:
Fixed a link here. Apparently, we can just omit the slash and it works. :shrug: 
Found this [here](https://stackoverflow.com/a/43894240/6795416).

Reply:
Oh didn't notice that. thanks.